README.md

```diff
@@ -150,22 +150,34 @@ When learning CS, there are some useful sites you must know to get always inform
+- [AngularJS Style Guide - Google](https://google.github.io/styleguide/angularjs-google-style.html) : An AngularJS Style Guide for Closure Users at Google
+- [Angular 1 Style Guide](https://github.com/johnpapa/angular-styleguide/tree/master/a1) : Officially endorsed style guide by John Papa
+- [Aurelia Style Guide](https://github.com/behzad888/Aurelia-styleguide) : An Aurelia style guide by Behzad Abbasi(Behzad888)
+- [C/C++ Coding Style for Emacs - Google](https://raw.githubusercontent.com/google/styleguide/gh-pages/google-c-style.el) : Emacs settings file for Google's C/C++ style
+- [(Good) C programming habits. • /r/C_Programming](https://www.reddit.com/r/C_Programming/comments/1vuubw/good_c_programming_habits/) : C programming habits to adopt
+- [C++ Style Guide - Google](https://google.github.io/styleguide/cppguide.html)
+- [C++ Style Compliance Tool - Google](https://github.com/google/styleguide/tree/gh-pages/cpplint) : This is automated checker to make sure a C++ file follows Google's C++ style guide
+- [CS 106B Coding Style Guide](http://stanford.edu/class/archive/cs/cs106b/cs106b.1158/styleguide.shtml) : Must see for those who create spaghetti
 - [Debugging Faqs](http://www.umich.edu/~eecs381/generalFAQ/Debugging.html) : Check out how to debug your program
-- [Directory of CS Courses (many with online lectures)](https://github.com/prakhar1989/awesome-courses) : Another online CS courses
-- [Directory of Online CS Courses](https://github.com/ossu/computer-science) : Free online CS courses
-- [Good C programming habits. • /r/C_Programming](https://www.reddit.com/r/C_Programming/comments/1vuubw/good_c_programming_habits/) : C programming habits to adopt
-- [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html)
+- [Google Style Guides](http://google.github.io/styleguide/) : Links to style guides and associated tools for many languages/environments.
 - [How to Report Bugs Effectively](https://www.chiark.greenend.org.uk/~sgtatham/bugs.html) : Want to report a bug but you don't how? Check out this post
-- [What are some bad coding habits you would recommend a beginner avoid getting into?](https://www.reddit.com/r/learnprogramming/comments/1i4ds4/what_are_some_bad_coding_habits_you_would/) : Bad habits to avoid when you get start
-- [PEP8 - Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008/) : Style Guide for Python Code
-- [Standard JS Style Guide](https://standardjs.com) : JavaScript style guide, with linter & automatic code fixer
-- [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html) : Google Python Style Guide
-- [Aurelia Style Guide](https://github.com/behzad888/Aurelia-styleguide) : An Aurelia style guide by Behzad Abbasi(Behzad888)
+- [HTML/CSS Style Guide - Google](https://google.github.io/styleguide/htmlcssguide.html) : Google HTML/CSS Style Guide
```

Comment:
Could these google language specific guides be sub-bullets under the "Google Style Guides" section above?  That might be a little clearer.